src/CoreWCF.NetTcp/src/CoreWCF/Configuration/ServiceModelWebHostBuilderExtensions.cs

```diff
@@ -101,7 +101,10 @@ private void OnServiceBuilderOpening(object sender, EventArgs e)
             foreach (ListenOptions listenOptions in ListenOptions)
             {
                 IPEndPoint endpoint = listenOptions.IPEndPoint;
-                var baseAddress = new Uri($"net.tcp://{endpoint.Address}:{endpoint.Port}/");
+                string address = endpoint.Address.ToString();
```

Comment:
This makes sense due to Uri understandable not being able to parse `net.tcp://:::805/` as a valid Uri. 😁